pkg/session/bootstrap.go

```diff
@@ -296,6 +296,7 @@ const (
 		source VARCHAR(10) NOT NULL DEFAULT 'unknown',
 		sql_digest varchar(64) DEFAULT NULL,
 		plan_digest varchar(64) DEFAULT NULL,
+		last_used_time TIMESTAMP DEFAULT NULL,
```

Comment:
Please also specify the precision, @henrybw just updated the precision to 6 to fix some unstable problems: https://github.com/pingcap/tidb/pull/63524/files